linux_os/guide/system/network/network-ufw/set_ufw_loopback_traffic/rule.yml

```diff
@@ -0,0 +1,43 @@
+documentation_complete: true
```

Comment:
```suggestion
documentation_complete: true

prodtype: ubuntu2004,ubuntu2204
```

Reply:
Could you include `prodtype`, please? Then we limit this rule to not appear in other data streams.